### PR TITLE
Issue #13693: migrate to coding to property macros

### PR DESCRIPTION
**Pull request by romani**
Issue #13693

### CODE REVIEW

File: src/xdocs/checks/coding/multiplestringliterals.xml [hunk 1]
```diff
@@ -32,35 +32,21 @@
...
-              </td>
-              <td>
-                subset of tokens
-                <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html">TokenTypes</a>
-              </td>
-              <td>
-                <code>ANNOTATION</code>
-              </td>
+              <td>Specify token type names where duplicate strings are ignored even if they don't match ignoredStringsRegexp. This allows you to exclude syntactical contexts like annotations or static initializers from the check.</td>
+              <td><a href="../../property_types.html#subset of tokens TokenTypes">subset of tokens TokenTypes</a></td>
```
Comment (nrmancuso) [inline src/xdocs/checks/coding/multiplestringliterals.xml:43]: CI is failing on this

Comment (romani) [reply src/xdocs/checks/coding/multiplestringliterals.xml:43]: Yes this is known issue, let me exclude this module from PR.

Comment (romani) [reply src/xdocs/checks/coding/multiplestringliterals.xml]: fixed

File: src/xdocs/checks/coding/returncount.xml [hunk 2]
```diff
@@ -48,45 +48,41 @@
               <td>Specify method names to ignore.</td>
               <td><a href="../../property_types.html#Pattern">Pattern</a></td>
               <td><code>&quot;^equals$&quot;</code></td>
-              <td>3.4</td>
```
Comment (nrmancuso) [inline src/xdocs/checks/coding/returncount.xml:51]: What happened here?

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: src/xdocs/checks/coding/returncount.xml (hunk 2 above)
Comment (romani) [reply src/xdocs/checks/coding/returncount.xml:51]: 3.4 -> 3.2
I doubt we will discover what is right value.
Check is from 3.2 https://github.com/checkstyle/checkstyle/blob/7e668593d0f88e6a549d71e0db931d2b1b16687c/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java#L98-L102

Property
https://github.com/checkstyle/checkstyle/blob/7e668593d0f88e6a549d71e0db931d2b1b16687c/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java#L156-L158

So 3.4 is better value.

Comment (romani) [reply src/xdocs/checks/coding/returncount.xml:51]: Let's address it separately https://github.com/checkstyle/checkstyle/issues/13934
As this version is so old, I propose to allow regression in and not block migration until this issue.
And I think I should pause migration of others and fix all issues we already found.

**Review by nrmancuso — APPROVED** [review]

**Pull request merged**